### PR TITLE
add basic express server app

### PR DESCRIPTION
**Pull request by jkronillis**

### CODE REVIEW

**Comment by efleming969** [comment]
are you crazy!

File: index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+var express = require("express");
+
+var server = express()
+server.get("/", function(req, resp) {
+	resp.send("happy time");
```
Comment (efleming969) [inline index.js:5]: should be 2 space and not 4

**Comment by eaharmsen** [comment]
42k files! ftw! :hankey:

File: index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+})
+server.listen(8080, function() {
+	console.log("server started");
```
Comment (vammar) [inline index.js:8]: The like to put the port the server is listening on as well in the console message. "server started on port 8080"

**Pull request closed without merging**